### PR TITLE
Webp Image Compression

### PR DESCRIPTION
**Pull request by mishuagopian**
# Summary 

### Webp Image Compression
Added optional image compression so that build folder is lighter

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,6 +67,33 @@ module.exports = [
...
+If you want to reduce your assets weight so that the build is ligther, you can turn on image compression option. To make this possible, set `imageCompression` variable as `true` in `gulp/config.js` file.
+
+
+##### Webp Compression
+
+Image compression is accomplished using [gulp-webp](https://github.com/sindresorhus/gulp-webp).
+
+Things to take into account:
+- If your assets include `.png` files, make sure `libpng` library is already installed.
+- If your assets include `.jpeg` files, make sure `libpng` library is already installed.
```
Comment (mishuagopian) [inline README.md]: Should change to libjpeg

File: README.md [hunk 2]
```diff
@@ -67,6 +67,33 @@ module.exports = [
...
+- If your assets include `.png` files, make sure `libpng` library is already installed.
+- If your assets include `.jpeg` files, make sure `libjpeg` library is already installed.
+
+If any of these is not installed you have to first uninstall `webp` like the following:
+```
+npm remove gulp-webp
+```
+Now, install the missing libraries.
+
+Finally, reinstall `gulp-webp` like the following:
```
Comment (sbalay) [inline README.md]: Finally, reinstall `gulp-webp` **with** the following command:

File: README.md [hunk 3]
```diff
@@ -67,6 +67,33 @@ module.exports = [
...
+
+##### Webp Compression
+
+Image compression is accomplished using [gulp-webp](https://github.com/sindresorhus/gulp-webp).
+
+Things to take into account:
+- If your assets include `.png` files, make sure `libpng` library is already installed.
+- If your assets include `.jpeg` files, make sure `libjpeg` library is already installed.
+
+If any of these is not installed you have to first uninstall `webp` like the following:
```
Comment (sbalay) [inline README.md]: **with** the following

Comment (sbalay) [inline README.md]: If any of these **are** not ...

**Comment by sbalay** [comment]
sunglasses

**Pull request merged**